plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -2346,7 +2346,7 @@ void producer_plugin_impl::produce_block() {
    }
 
    //idump( (fc::time_point::now() - chain.pending_block_time()) );
-   block_state_ptr pending_blk_state = chain.finalize_block( [&]( const digest_type& d ) {
+   block_state_ptr pending_blk_state = chain.finalize_block( [relevant_providers]( const digest_type& d ) {
```

Comment:
`relevant_providers` can be moved into the lambda.

Reply:
Done.